deploy.ci.acceptance.robot.yml

```diff
@@ -51,7 +51,7 @@ groups:
                         region: EU
                         services:
                             session:
-                                namespace: 1
+                                namespace: 2
```

Comment:
Why is it changed?

Reply:
Actually, it was not. GitHub shows the difference falsy. 

`deploy.ci.acceptance.robot.yml` was created from `deploy.ci.acceptance.yml` but it shows as `deploy.ci.api.yml` renaming which is not true. 

And if you check the original `deploy.ci.acceptance.yml` the namespace ids are valid.